core/src/main/java/org/opensearch/sql/expression/datetime/DateTimeFormatterUtil.java

```diff
@@ -201,6 +254,90 @@ static ExprValue getFormattedTime(ExprValue timeExpr, ExprValue formatExpr) {
     return getFormattedString(formatExpr, TIME_HANDLERS, time);
   }
 
+  private static boolean canGetDate(TemporalAccessor ta) {
+    if (!ta.isSupported(ChronoField.YEAR)
+        || !ta.isSupported(ChronoField.MONTH_OF_YEAR)
+        || !ta.isSupported(ChronoField.DAY_OF_MONTH)) {
+      return false;
+    }
+    return true;
```

Comment:
Optional: Could be simplified a bit (the same for `canGetTime`)
```suggestion
    return ta.isSupported(ChronoField.YEAR)
        && ta.isSupported(ChronoField.MONTH_OF_YEAR)
        && ta.isSupported(ChronoField.DAY_OF_MONTH));
```

Reply:
Fixed in https://github.com/opensearch-project/sql/pull/1420/commits/f21517e921222d001e06ed93d940fcb12c481b02